app/org/sagebionetworks/bridge/services/SchedulePlanService.java

```diff
@@ -43,8 +43,9 @@ public final void setSurveyService(SurveyService surveyService) {
         this.surveyService = surveyService;
     }
 
-    public List<SchedulePlan> getSchedulePlans(ClientInfo clientInfo, StudyIdentifier studyIdentifier) {
-        return schedulePlanDao.getSchedulePlans(clientInfo, studyIdentifier);
+    public List<SchedulePlan> getSchedulePlans(ClientInfo clientInfo, StudyIdentifier studyIdentifier,
+            boolean includeDeleted) {
+        return schedulePlanDao.getSchedulePlans(clientInfo, studyIdentifier, includeDeleted);
```

Comment:
It looks like we have tests for SchedulePlanDao.getSchedulePlans(), but not for SchedulePlanService.getSchedulePlans().

Reply:
Added tests for these.